content/docs/jsx-in-depth.md

```diff
@@ -247,6 +247,39 @@ function App2() {
 
 Spread attributes can be useful when you are building generic containers. However, they can also make your code messy by making it easy to pass a lot of irrelevant props to components that don't care about them. We recommend that you use this syntax sparingly.
 
+An alternative to spreading all of your props is to specifically list out all the properties that you would like to consume, followed by `...other`.
+
+```js
+const { children, ...other } = props;
+```
+
+This ensures that you pass down all the props *except* the ones you're consuming yourself.
+
+```js{6}
```

Comment:
I didn't know what this markdown syntax was before, changed it to highlight line 6 like you suggested.